README.md

```diff
@@ -375,6 +376,20 @@ fmt.Println(output)
 // Output: contents of file
 ```
 
+## ListFiles
+
+`ListFiles()` Creates a pipe with list of files under the given path or glob.
```

Comment:
How about something like this:

`ListFiles()` lists files, like Unix [`ls`](examples/ls/main.go). It creates a pipe containing all files and directories matching the supplied path specification, one per line. This can be the name of a directory (`/path/to/dir`), the name of a file (`/path/to/file`), or a _glob_ (wildcard expression) conforming to the syntax accepted by [filepath.Match](https://golang.org/pkg/path/filepath/#Match) (`/path/to/*`).

```go
p := script.ListFiles("/tmp/*.php")
files, err := p.String()
if err != nil {
	log.Fatal(err)
}
fmt.Println("found suspicious PHP files in /tmp:")
fmt.Println(files)
```

Reply:
Don't forget to add `ls` to the list of examples at the end of the README.

Reply:
I've copied your exact description to readme. I am really bad at this, so if there are any issues with readme/docs it would be great if you can fix them.